src/components/PostList/PostList.tsx

```diff
@@ -0,0 +1,28 @@
+import { PostInfo } from '../PostInfo';
+import './PostList.scss';
+import { Post } from '../../types/Post';
+
+type Props = {
+  posts: Post[]
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <ul className="posts">
+    {
+      posts.map(post => (
+        <li key={post.id} className="posts__post">
+          {
+            post.user && (
+              <PostInfo
+                user={post.user}
+                comments={post.comments}
+                title={post.title}
+                body={post.body}
+              />
+            )
```

Comment:
You're rendering whole post info based on user's value in this post, that's doesn't seem right. Anyway after that you're checking user's value inside this component.